.editorconfig

```diff
@@ -1,5 +1,16 @@
 root = true
 
-[*.js]
-indent_style=tab
-trim_trailing_whitespace=true
\ No newline at end of file
+[*]
+indent_style = tab
+indent_size = 2
+end_of_line = lf
+charset = utf-8
```

Comment:
Done in [this commit](https://github.com/webpack/webpack/pull/2849/commits/84138c8de191a39cd418f4e1d2f2a6449316cf8f)